dlog/kimchi/src/bench.rs

```diff
@@ -84,35 +107,33 @@ pub fn proof(num: usize) {
 
         // previous opening for recursion
         let prev = {
-            let k = ceil_log2(index.srs.g.len());
+            let k = ceil_log2(self.index.srs.g.len());
             let chals: Vec<_> = (0..k).map(|_| Fp::rand(rng)).collect();
             let comm = {
                 let coeffs = b_poly_coefficients(&chals);
                 let b = DensePolynomial::from_coefficients_vec(coeffs);
-                index.srs.commit_non_hiding(&b, None)
+                self.index.srs.commit_non_hiding(&b, None)
```

Comment:
does it make sense to also have benchmark with hiding commitments?

Reply:
I'm ignoring the recursion stuff